app/forms/steps/client/benefit_type_form.rb

```diff
@@ -37,7 +37,8 @@ def persist!
 
       def attributes_to_reset
         {
-          'last_jsa_appointment_date' => (last_jsa_appointment_date if jsa?)
+          'last_jsa_appointment_date' => (last_jsa_appointment_date if jsa?),
+          'has_benefit_evidence' => nil,
```

Comment:
As `has_benefit_evidence` is being reset in the benefit type form, I believe this makes resetting the attribute on the dwp check redundant as this screen will always be passed before the dwp check screen